ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -274,7 +280,10 @@ export class FrontstageDef {
     }
 
     for (const popoutId of popoutsToOpen) {
-      this.openPopoutWidgetContainer(popoutId);
+      if (oldState) {
+        const result = this.openPopoutWidgetContainer(popoutId, oldState);
+        return result;
```

Comment:
Do we assume only a single popout can be opened at a time (return in a loop)?

Reply:
I was unsure about the mechanics of this. It seemed like it is setup to be able to open multiple at a time, but I wasn't sure if it was possible for the user to open multiple at a time. Because they have to click the button for the widget to popout, so it didn't seem like it to me. Can change if it is possible

Reply:
Looks like we need to set up some testing for electron...
This should be possible with electron (once again on initialization, but in any handler as well).

Reply:
I.e. by calling https://www.itwinjs.org/reference/appui-react/frontstage/frontstagedef/popoutwidget/

Reply:
Yeah, I think we could get away with a couple of unit tests:
- Simulate popout button click (i.e. just dispatch an action)
- Simulate "electron" initialization w/ multiple popouts (not sure if there are cases where the window would fail to open on the electron)

Assert on opened windows, widget state changes, and frontstageDef.nineZoneState.

Reply:
We already have a simulated popout button dispatch test. The test is located in `FrontstageDef.test.tsx`. I'll create a new one for initialization for multiple popouts

Reply:
New test has been created to cover the second scenario